recipe/meta.yaml

```diff
@@ -63,3 +63,4 @@ extra:
   recipe-maintainers:
     - ZhuozhaoLi
     - yadudoc
+    - xylar
```

Comment:
I'm happy to *not* add myself as a maintainer if you prefer to take care of this package yourselves.  Just volunteering to help if you want it.

Reply:
I'm happy you added yourself, we wish more of the contributors do this more often.